src/vm/runners/cairo_runner.rs

```diff
@@ -94,6 +94,10 @@ impl RunResources {
     pub fn consume_steps(&mut self) {
         self.n_steps -= 1;
     }
+
+    pub fn get_n_steps(&mut self) -> usize {
```

Comment:
why do we need a mutable reference here?

Reply:
Thanks, good catch!
copy paste staff